pkg/operator/ceph/version/version.go

```diff
@@ -41,6 +42,12 @@ func (v *CephVersion) String() string {
 		v.Major, v.Minor, v.Extra, v.ReleaseName())
 }
 
+// StringHuman returns the Ceph version in a human readable format
+func (v *CephVersion) StringHuman() string {
```

Comment:
Is the format in order to match the output format of `ceph version` so the version can be parsed in the same way? Then the purpose isn't to be human-readable, so maybe we just call the method `String()`?

Reply:
I think that we should have `String()` be the free-form version which also conforms to the `Stringer` interface and is more casual / used in logging, and make `StringHuman` be something like `CephVersionFormatted` or some such naming--its format is special as its directly consumed by the parser creating a dependency on its format through the environment variable--that path can be documented as a docstring on the method or something.